### PR TITLE
#161913533: update frontend URL

### PR DESCRIPTION
**Pull request by uchemukolo**
#### What does this PR do?
- Redirect social authentication to use frontend URL
#### Description of Task to be completed?
- Update test social services test files
- Add redirect URL
- Update base URL in the mail.js file to use frontend URL
#### How should this be manually tested?
- Fetch and checkout to the branch
```$xslt
> git fetch
> git checkout ft-161913533-update-frontend-url

- Setup Base Url using .env-sample

 ```
- Run application
```$xslt 
> npm run start
```
- To run test
```$xslt 
> npm run test
```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161913533](<https://www.pivotaltracker.com/story/show/161913533>)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: controllers/auth/AuthController.js [hunk 1]
```diff
@@ -8,6 +9,10 @@ import eventBus from '../../helpers/eventBus';
 import HttpError from '../../helpers/exceptionHandler/httpError';
 import Util from '../../helpers/Util';
 
+dotenv.config();
+
+console.log('>>>>', process.env.NODE_ENV);
```
Comment (omedale) [inline controllers/auth/AuthController.js]: Remove this please

File: tests/routes/facebookLogin.test.js [hunk 2]
```diff
@@ -13,14 +13,16 @@ describe('GET /auth/facebook', () => {
...
-        expect(res).to.have.status(200);
-        expect(res.body.user).to.have.property('token');
-        expect(res.body.user.token).to.not.be.null;
-        expect(res.body.user).to.have.property('imageUrl');
-        expect(res.body.user).to.have.property('bio');
-        expect(res.body.user)
-          .to.have.property('email')
-          .that.is.equal('johndoe@gmail.com');
+        expect(res).to.redirect;
+        // expect(res).to.redirectTo
```
Comment (omedale) [inline tests/routes/facebookLogin.test.js]: Why do we need to comment this out

File: tests/routes/googleLogin.test.js [hunk 3]
```diff
@@ -13,14 +13,15 @@ describe('GET /auth/google', () => {
...
-        expect(res).to.have.status(200);
-        expect(res.body.user).to.have.property('token');
-        expect(res.body.user.token).to.not.be.null;
-        expect(res.body.user).to.have.property('imageUrl');
-        expect(res.body.user).to.have.property('bio');
-        expect(res.body.user)
-          .to.have.property('email')
-          .that.is.equal('johndoe@gmail.com');
+        expect(res).to.redirect;
+        // expect(res).to.have.status(200);
```
Comment (omedale) [inline tests/routes/googleLogin.test.js]: Please remove this code if we don't need it anymore

File: tests/routes/twitterLogin.test.js [hunk 4]
```diff
@@ -13,14 +13,15 @@ describe('GET /auth/twitter', () => {
...
-        expect(res.body.user).to.have.property('token');
-        expect(res.body.user.token).to.not.be.null;
-        expect(res.body.user).to.have.property('imageUrl');
-        expect(res.body.user).to.have.property('bio');
-        expect(res.body.user)
-          .to.have.property('email')
-          .that.is.equal('johndoe@gmail.com');
+        expect(res).to.redirect;
+        // expect(res).to.have.status(200);
+        // expect(res.body.user).to.have.property('token');
```
Comment (omedale) [inline tests/routes/twitterLogin.test.js]: Also remove this

Comment (uchemukolo) [reply tests/routes/twitterLogin.test.js]: All commented code has been removed.

File: tests/routes/googleLogin.test.js (hunk 3 above)
Comment (uchemukolo) [reply tests/routes/googleLogin.test.js]: Removed, not needed anymore.

File: tests/routes/facebookLogin.test.js (hunk 2 above)
Comment (uchemukolo) [reply tests/routes/facebookLogin.test.js]: It is not needed anymore

File: controllers/auth/AuthController.js (hunk 1 above)
Comment (uchemukolo) [reply controllers/auth/AuthController.js]: All commented code has been removed.

**Pull request merged**